homeassistant/components/axis/device.py

```diff
@@ -179,7 +179,7 @@ def shutdown(self, event):
     import axis
 
     device = axis.AxisDevice(
-        loop=hass.loop, host=config[CONF_HOST],
```

Comment:
This would break the Axis integration since it expects loop to be passed in right now

Reply:
An async library should be prepared to accept `None` at that point. Care to file a bug with them?

Reply:
It is only for hass and that hasn't been an issue so far. Need to get the current Axis PR merged in order to change this

Reply:
The small handful of these can be found by not removing loop if it's not the last parameter. There's only three I could see.

Reply:
My bad. I tried to make sure I didn't hit any of the classes 🤷‍♂